src/llmcompressor/entrypoints/README.md

```diff
@@ -0,0 +1,53 @@
+# LLM Compressor Entrypoints
+
+## Oneshot
+
+Model optimizations compress models while preserving accuracy. One-shot in LLM-Compressor supports faster inference on vLLM by applying post-training quantization (PTQ) or sparsification
+
```

Comment:
Let's focus this readme on the components/steps in OneShot.  We can take most of those details from the docstring you've added.